nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -101,6 +107,30 @@ public Document(String id, String type, List<String> facets, String changeToken,
         this.dirtyProperties = new HashMap<>();
     }
 
+    /**
+     * @since 2.3
+     */
+    public Document(String id, String type, List<String> facets, String changeToken, String path, String state,
+            String lockOwner, String lockCreated, String repositoryName, String versionLabel, String isCheckedOut,
+            Map<String, Object> properties, Map<String, Object> contextParameters, String isProxy) {
```

Comment:
I think it's better to have the `isProxy` argument after the `isCheckedOut` one to ease the readability. Moreover, `isProxy` should be a `boolean` (or `Boolean` if really needed?).